sphinx_intl/catalog.py

```diff
@@ -23,7 +23,7 @@ def load_po(filename):
         return pofile.read_po(f, charset=charset)
 
 
-def dump_po(filename, catalog, line_width=76):
+def dump_po(filename, catalog, line_width=76, sort_output=False, sort_by_message_location=False):
```

Comment:
Please add param lines on docstring for new 2 parameters.

These two arguments are not set to True within this tool and may be removed as unnecessary arguments. As a remedy, please describe the purpose for which they are arguments in the docstring or code comment.